src/test/steps/common.ts

```diff
@@ -221,18 +222,24 @@ When('I enter my valid case reference and valid access code', async () => {
 });
 
 When('a case worker issues the application', async () => {
-  await triggerCaseWorkerEvent('caseworker-issue-application', { ceremonyPlace: 'Somewhere' });
+  await triggerAnEvent('caseworker-issue-application', { ceremonyPlace: 'Somewhere' });
```

Comment:
just preference but this could be called 'triggerEvent'

Reply:
There's already another function with this name so will stick with triggerAnEvent.